faux-mgs/src/main.rs

```diff
@@ -373,6 +387,14 @@ enum Command {
 
     /// Reads the lock status of any VPD in the system
     VpdLockStatus,
+
+    /// Read the RoT's boot-time information.
+    RotBootInfo {
+        /// Return highest version of RotBootInfo less then or equal to our
+        /// highest known version.
+        #[clap(long, short, default_value = formatcp!("{}", RotBootInfo::HIGHEST_KNOWN_VERSION))]
```

Comment:
Nit - we don't need const formatting; this can be

```suggestion
        #[clap(long, short, default_value_t = RotBootInfo::HIGHEST_KNOWN_VERSION)]
```

as long as the type matches. (It looks like `HIGHEST_KNOWN_VERSION` is a `u32` - it should be a `u8`, right?)

Reply:
right. done